ckanext/datagovuk/schema.py

```diff
@@ -33,4 +33,14 @@ def show_package_schema(schema_dict):
                 toolkit.get_validator('ignore_missing')]
         })
 
-    return schema_dict
\ No newline at end of file
+    return schema_dict
+
+def user_new_form_schema():
+    from ckan.logic.schema import user_new_form_schema as default_user_new_form_schema
```

Comment:
It's okay for now, but we don't really have to do the imports here, they might be better at the top.  We did this historically but I'm not sure it's a great idea.  

One for next refactor I think.